### ISSUE TITLE
Auto Updating Build 3801 No Longer Working - Causes Update Loop 

### ISSUE THREAD

**Issue by MattyXarope** [pineappleEA/pineapple-src#98]
The latest (afaik) build that auto updates, build 3801, prompts the user to update but fails and gets constantly stuck in a loop. 

The terminal output is as follows: 

```AppImageUpdate version 1-alpha (commit cb611d8), build 431 built on 2020-04-07 21:47:19 UTC

Fetching release information for tag "continuous" from GitHub API.

Updating from GitHub Releases via ZSync

zsync2: /home/deck/Applications/yuzu-x86_64.AppImage found, using as seed file

zsync2: Target file: /home/deck/Applications/yuzu-x86_64.AppImage

zsync2: Reading seed file: /home/deck/Applications/yuzu-x86_64.AppImage

zsync2: Usable data from seed files: [100.000000](tel:100000000)%

zsync2: Renaming temp file

zsync2: Fetching remaining blocks

zsync2: Verifying downloaded file

zsync2: checksum matches OK

zsync2: used 86374400 local, fetched 0

qt.qpa.xcb: QXcbConnection: XCB error: 3 (BadWindow), sequence: 765, resource id: 16777525, major code: 40 (TranslateCoords), minor code: 0

(zenity:21047): GLib-GObject-CRITICAL **: 19:21:46.801: g_object_unref: assertion 'G_IS_OBJECT (object)' failed```

**Comment by MGThePro** [comment]
Yeah, the auto updating builds are gone and wont return. I don't know a of a way to present some kind of notice that it no longer works, but if you do, feel free to suggest it.

**Comment by linh1987** [comment]
@MGThePro any reasoning behind it? would it help if somebody volunteers to take on that functionalities?

**Comment by MGThePro** [comment]
Two reasons mostly

The first is that the source code that is provided isn't complete enough anymore to build it. Some of the submodule folders are just empty and we cant push the git files to just redownload them on the CI. (see issue #97 and #96 ). Sure it could be done with some elbow grease but that leads me to my second point.

There's no point in building it for linux anymore since there are now official linux appimages that are practically guaranteed to work. The previous appimages that were built on our CI had a lot of problems (see issues #93 #88 #85 #95 #79). All of that can be fixed probably but I'm not able to invest this much time into it. And even if someone else was trying to fix all the issues there's a good chance it would be abandoned again after a few months.

**Comment by EtereosDawn** [comment]
this is sad... it was good while the updates lasted.

**Comment by XDM-Inc** [comment]
Same here, no more auto updating. But also for me it dosent even check for updates either now.

**Comment by XDM-Inc** [comment]
well if auto updating is done for can we at least make it that when we download a new build it has a unified name instead of build numbers? then we can easily copy the manually download appimage as a drop in replacement without having to rename every time and also start working on a new method of updating

**Comment by XDM-Inc** [comment]
#105 If anyone knows a bit about scripts, im trying to get a rudimentary downloader working

**Comment by Beaverbroo** [comment]
does someone know what to do with the updater? Do you just leave the file in the downloads or you need to put it inside yuzu?

**Comment by Beaverbroo** [comment]
cause for me it doesnt work either but i dont get the prompt

**Comment by MGThePro** [comment]
> does someone know what to do with the updater? Do you just leave the file in the downloads or you need to put it inside yuzu?

If you mean the auto updating appimage for linux, delete it. It's not getting updates anymore and it wont ever get any
